tsconfig.json

```diff
@@ -32,5 +34,5 @@
     ".contentlayer/generated",
     ".contentlayer/generated/**/*.json"
   ],
-  "exclude": ["node_modules", ".contentlayer"]
+  "exclude": ["node_modules", ".contentlayer", "public/tag-data.json"]
```

Comment:
any specific reason for excluding `"public/tag-data.json"`? 
we might as well exclude the whole directory `public`

Reply:
Oh, yh